jekyll/_cci2_ja/orbs-faq.md

```diff
@@ -61,7 +61,7 @@ contentTags:
 
 * **回答:** Orb には[セマンティックバージョニング]()が使用されています。つまり、_メジャー_バージョンを設定すると (例:`3` )、_マイナー_と_パッチ_のすべてのアップデートを受け取ります。静的にバージョンを設定すると(例:`3.0.0` )、アップデートは適用されません。これが最も決定論的で推奨される方法です。
 
-_**注: **<非推奨> `@volatile` を使って最新バージョンの Orb を受け取ることも可能です。 これは互換性を損なう変更が含まれる場合があるため推奨していません。_
+注: **非推奨** `@volatile` を使って最新バージョンの Orb を受け取ることも可能です。 これは互換性を損なう変更が含まれる場合があるため推奨していません。
```

Comment:
Thanks @yuki-sakamoto-san for this fix! :)

Reply:
![Screen Shot 2023-03-16 at 12 58 27 PM](https://user-images.githubusercontent.com/1449325/225738782-ac49349a-eb48-4c8e-a880-f7a478aaae8e.png)